ui/src/common/actions.ts

```diff
@@ -605,16 +605,23 @@ export const StateActions = {
     // rather than T1, T10, T11, ..., T2, T20, T21 .
     const coll = new Intl.Collator([], {sensitivity: 'base', numeric: true});
     for (const group of Object.values(state.trackGroups)) {
-      group.tracks.sort((a: string, b: string) => {
-        const aRank = getFullKey(a);
-        const bRank = getFullKey(b);
-        for (let i = 0; i < aRank.length; i++) {
-          if (aRank[i] !== bRank[i]) return aRank[i] - bRank[i];
-        }
+      // This sort is not working as intended
```

Comment:
When just looking at the new code and not being aware of the change, the comment is a bit odd, unexpected, no?
It states

// This sort is not working as intended

but I hope the new code using group.sortOrder does work as intended :-).